packages/es-dev-server/src/create-middlewares.ts

```diff
@@ -129,6 +137,12 @@ export function createMiddlewares(
     );
   }
 
+  let cache: LRUCache<string, CacheEntry> | undefined;
```

Comment:
I think we can move these inside `createPluginTransformMiddlware` since that's the only consumer.

Reply:
Yep :) true, not really sure why I did it but I moved them to `createPluginTransformMiddleware` now and it made the code cleaner for sure